packages/amazonq/src/lsp/chat/messages.ts

```diff
@@ -353,6 +363,24 @@ export function registerMessageListeners(
             params: params,
         })
     })
+
+    languageClient.onNotification(openFileDiffNotificationType.method, async (params: OpenFileDiffParams) => {
+        const uri = vscode.Uri.parse(params.originalFileUri)
```

Comment:
why can't we just use viewDiff from contentController? is there something missing there?

Reply:
I was trying to avoid using the message thing but actually let me give it a try